api/src/libs/messaging.ts

```diff
@@ -46,10 +45,55 @@ export const contactsForProfile = async (profileId: number): Promise<string[]> =
   }
 }
 
+export const profileDetails = async (profileId: number): Promise<string> => {
+
+  try {
+    const dm = new DataManager(shared.pgPool);
+    const { ProfileModel } = dm;
+    const profiles = await ProfileModel.findById(profileId);
+    const profileName = profiles.name;
+    return profileName;
```

Comment:
Could probably do with one space in here just before the return. I think the only advantage would be to break the code up into "stanzas" for lack of a better work to improve readability.